traffic_control/clients/README.md

```diff
@@ -8,8 +8,11 @@ There are two client libraries supported:
 * [Documentation](https://github.com/apache/trafficcontrol/tree/master/traffic_control/clients/python/trafficops)
 
 ## Golang
-### TO API v1.5 _(Deprecated)_
-* [Documentation](https://github.com/apache/trafficcontrol/tree/master/traffic_ops/v1-client)
+### TO API v2 _(Deprecated)_
+* [Documentation](https://github.com/apache/trafficcontrol/tree/master/traffic_ops/client)
+
+### TO API v3 _(Deprecated)_
```

Comment:
TO API v3 is deprecated? I thought it's the "stable" version until API v2 is removed and API v5 is added.

Reply:
> TO API v3 is deprecated? I thought it's the "stable" version until API v2 is removed and API v5 is added.

Reply:
My understanding was that versions 2 and 3 are deprecated once 4 is released Then with ATCv7 we can remove them and deprecate 4 assuming 5 was under development after ATCv6's release and released with ATCv7 as the new stable API.

Reply:
That makes sense, but in order to remove them API v2 and API v3 in ATC 7, a separate PR will need to mark all of the endpoints as deprecated in the documentation for ATC 6.